app/helper/rtForm.php

```diff
@@ -662,6 +722,31 @@ public function display_link( $args = '' ) {
 			echo $this->get_link( $args );
 		}
 
+		/**
+		 * Get rtmedia html input file in admin options.
+		 *
+		 * @access public
+		 *
+		 * @param string|array $attributes Arguments to create file input control.
+		 *
+		 * @return string
+		 */
+		public function get_inputfile( $attributes = '' ) {
+			return $this->generate_inputfile( $attributes );
+		}
+
+
+		/**
+		 * Display file input settings for set custom thumbnail generator.
+		 *
+		 * @param string|array $args Arguments to create file input control.
+		 *
+		 * @throws rtFormInvalidArgumentsException Invalid argument exception.
+		 */
+		public function display_inputfile( $args = '' ) {
+			echo $this->get_inputfile( $args );
```

Comment:
add comment above `Previously escaped and sanitized so not required here.`
add a comment on this line `// WPCS: XSS ok.`